.eslintignore

```diff
@@ -11,3 +11,5 @@ node_modules
 pnpm-lock.yaml
 package-lock.json
 yarn.lock
+/src/lib/language_servers/*
```

Comment:
looks like these /src/lib/ stuff is still from before the language server worker was it's own module, right? 👀